.github/workflows/pipeline.yml

```diff
@@ -0,0 +1,20 @@
+name: Continuous Integration
+on: 
+  push:
+    branches: 
+      - main
+    paths-ignore:
+      - "README.md"
+  pull_request:
+    paths-ignore:
+      - "README.md"
```

Comment:
I updated the README and pushed and the pipeline still ran on the pull request, did I get something wrong? [GitHub docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-excluding-paths) show this as an example
```
on:
  push:
    paths-ignore:
      - 'docs/**'
```